ambry-store/src/main/java/com.github.ambry.store/CompactionLog.java

```diff
@@ -67,19 +67,17 @@ static boolean isCompactionInProgress(String dir, String name) {
    * @param dir the directory at which the compaction log must be created.
    * @param name unique name of the store.
    * @param time the {@link Time} instance to use.
-   * @param compactionDetailsList the details about the compaction.
+   * @param compactionDetails the details about the compaction.
    */
-  CompactionLog(String dir, String name, Time time, List<CompactionDetails> compactionDetailsList) throws IOException {
+  CompactionLog(String dir, String name, Time time, CompactionDetails compactionDetails) throws IOException {
```

Comment:
Discussed this offline - this is due to the change in contract b/w the `CompactionManager` and the `DataMover`.